.travis.yml

```diff
@@ -76,7 +76,22 @@ deploy:
     tags: true #GH requires tagged release
     branch: master
     repo: htm-community/htm.core
-
+   
+  ## PYPI releases 
+  provider: pypi 
+  user: "__token__"   # token for PYPI test account for David Keeney.  Not for production
+  password: "pypi-AgENdGVzdC5weXBpLm9yZwIkOTk0YmZjNGYtZTgxNS00Yjk2LTg5ZTAtODE1MGI4MjZhNGZlAAIleyJwZXJtaXNzaW9ucyI6ICJ1c2VyIiwgInZlcnNpb24iOiAxfQAABiDXJOuxvodsEDoD5dOH-e0td1DdUSwrl2NCl_lP_vy6RA"
+  server: "https://test.pypi.org/legacy/"
+  on:
+    tags: true
```

Comment:
nit, for testing, you can try with tags: false. Maybe PYPI deploy will work with it.

Reply:
But will your shell script run with tags: false?

Reply:
not sure. The script will work w/o tags, and is run in `before_deploy`. The "releases" (GH) provider runs only on tags, but you're adding a second provider (that could run on non-tags)